packages/node_modules/@webex/internal-plugin-ediscovery/src/transforms.js

```diff
@@ -475,6 +517,21 @@ class Transforms {
       return Promise.resolve(object);
     }
 
+    // decrypt description if present with a descriptionEncryptionKeyUrl
+    if (container.description && container.descriptionEncryptionKeyUrl) {
```

Comment:
The container description uses a different key. This is what I recalled you saying at the scrum. I guess the activity keys are always the same. How come it can't occur for the `name` of the container? How come we don't check the `name` here but do use the `description`?

Actually I'm wrong, below we decrypt the name. Can you elaborate on when the keys would be different? i.e. Unlike with the activities where they are the same.

Reply:
According to Diego they may end up being different due to key rotation, there may be other reasons, it has apparently always been part of the design: https://confluence-eng-gpk2.cisco.com/conf/pages/viewpage.action?spaceKey=WBXT&title=Conversation%3A+Public+Spaces#Conversation:PublicSpaces-Description